src/main/java/org/jboss/pnc/logprocessor/eventduration/domain/LogEvent.java

```diff
@@ -135,6 +159,7 @@ public void addDuration(Duration duration) {
         public void configure(Map<String, ?> configs, boolean isKey) {
         }
 
+        @Timed
```

Comment:
When are this annotations processed as this class is not used an a bean?

Reply:
Should be processed at compile time and since @Timed is annotated by @Retention(RetentionPolicy.RUNTIME) the methods annotated by @Timed will record their execution time from start to return or till they throw an exception.
    ...